platformio/package/manifest/schema.py

```diff
@@ -142,9 +145,17 @@ class ManifestSchema(Schema):
         )
     )
 
-    def handle_error(self, error, data):
-        if self.strict:
-            raise ManifestValidationError(error, data)
+    # Wrapper to encapsulate differences between marshmallow 2 and 3
+    # load() method
+    def load_manifest(self, manifest):
+        if marshmallow_version >= (3, ):
+            data = self.load(manifest)
+        else:
+            data, _ = self.load(manifest)
+        return data
+
+    def handle_error(self, error, data, *, _many, **_kwargs):
```

Comment:
```suggestion
    def handle_error(self, error, data, _many, **_kwargs):
```

No keyword only arguments in Python 2.7